modules/keyframes.js

```diff
@@ -27,6 +27,7 @@ if (isAnimationSupported) {
 // Simple animation helper that injects CSS into a style object containing the
 // keyframes, and returns a string with the generated animation name.
 var keyframes = function (
+  component: any,
```

Comment:
I don't think keyframes should have to take this in; let`s just pass the component as null, since you may not have a component to pass (e.g. you export a bunch of keyframe animations from a module).